app/test/process_proposal_test.go

```diff
@@ -49,6 +50,26 @@ func TestProcessProposal(t *testing.T) {
 			testfactory.RandomBlobNamespaces(tmrand.NewRand(), 4),
 			[][]int{{100}, {1000}, {420}, {300}},
 		),
+		blobfactory.DefaultTxOpts()...,
+	)
+
+	largeMemo := strings.Repeat("a", appconsts.MaxTxBytes(appconsts.LatestVersion))
+
+	// create 2 single blobTxs that include a large memo making the transaction
+	// larger than the configured max tx bytes
+	largeBlobTxs := blobfactory.ManyMultiBlobTx(
+		t, enc, kr, testutil.ChainID, accounts[3:], infos[3:],
+		blobfactory.NestedBlobs(
+			t,
+			testfactory.RandomBlobNamespaces(tmrand.NewRand(), 4),
+			[][]int{{100}, {1000}, {420}, {300}},
+		),
+		user.SetMemo(largeMemo))
+
+	// create 1 large sendTx that includes a large memo making the
+	// transaction over the configured max tx bytes limit
+	largeSendTx := testutil.SendTxsWithAccounts(
+		t, testApp, enc, kr, 1000, accounts[0], accounts[1:2], testutil.ChainID, user.SetMemo(largeMemo),
 	)
```

Comment:
I think the memo size is capped (256 bytes IIRC) so this will fail regardless

Reply:
do you know where the memo check is? because it was failing with transaction too large error in the logs

Reply:
It's called `ValidateMemoDecorator`

Reply:
`NewMaxTxSizeDecorator` is called before so it'll be rejected for transaction being too large